components/WebServer/controllers/files/waveform.js

```diff
@@ -0,0 +1,36 @@
+const debug = require('debug')('linto:components:WebServer:controller:file:waveform')
```

Comment:
Maybe merge waveform and transformfile to handle child_process

Only the command of spawn process change on both usage no need for duplicate code